src/services/utils.js

```diff
@@ -3,4 +3,19 @@ const generateUniqeUsername = (email) => {
   return `${email.split('@')[0]}_${new Date().valueOf()}`;
 };
 
-module.exports = { generateUniqeUsername };
+// If I delete the next, the code is not working. Why?
+function errorHandler(err, req, res, next) {
+  if (err.name === 'UnauthorizedError') {
+    res.status(401).json({
+      error: true,
+      message: `Invalid Token: ${err.message}`,
+    });
+  } else {
+    res.status(500).json({
+      error: true,
+      message: 'Internal server error occured',
+    });
+  }
+}
+
+module.exports = { generateUniqeUsername, errorHandler };
```

Comment:
Can we add return statement to avoid any errors

Reply:
It's unnecessary since we have if and else written explicitly.